### PR TITLE
C100 rebuild base functional tests

### PR DESCRIPTION
**Pull request by yogendraU**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/test/end-to-end/codecept.conf.js [hunk 1]
```diff
@@ -25,8 +25,30 @@ exports.config = {
...
+    RespondentDetails: './pages/C100-Rebuild/RespondentDetails.js',
+    OtherPersonDetails: './pages/C100-Rebuild/OtherPersonDetails.js',
+    OtherProceedings: './pages/C100-Rebuild/OtherProceedings.js',
+    SafetyConcerns: './pages/C100-Rebuild/SafetyConcerns.js',
+    InternationElements: './pages/C100-Rebuild/InternationElement.js',
+    ReasonableAdjustments: './pages/C100-Rebuild/ReasonableAdjustments.js',
+    HelpWithFees: './pages/C100-Rebuild/HelpWithFees.js',
+    CheckYourAnswers: './pages/C100-Rebuild/CheckYourAnswers.js',
+    CheckYourAnswersSimple: './pages/C100-Rebuild/CheckYourAnswersSimple',
+    ConsentOrder: './pages/C100-Rebuild/ConsentOrder.js'
```
Comment (jahnavi1satya) [inline src/test/end-to-end/codecept.conf.js:48]: These are included in the steps_file, so probably not needed here

File: saucelabs.conf.js [hunk 2]
```diff
@@ -82,6 +82,25 @@ const setupConfig = {
...
+    RespondentDetails: './pages/C100-Rebuild/RespondentDetails.js',
+    OtherPersonDetails: './pages/C100-Rebuild/OtherPersonDetails.js',
+    OtherProceedings: './pages/C100-Rebuild/OtherProceedings.js',
+    SafetyConcerns: './pages/C100-Rebuild/SafetyConcerns.js',
+    InternationElements: './pages/C100-Rebuild/InternationElement.js',
+    ReasonableAdjustments: './pages/C100-Rebuild/ReasonableAdjustments.js',
+    HelpWithFees: './pages/C100-Rebuild/HelpWithFees.js',
+    CheckYourAnswers: './pages/C100-Rebuild/CheckYourAnswers.js',
+    CheckYourAnswersSimple: './pages/C100-Rebuild/CheckYourAnswersSimple',
+    ConsentOrder: './pages/C100-Rebuild/ConsentOrder.js'
```
Comment (jahnavi1satya) [inline saucelabs.conf.js:103]: These are included in the steps_file, so probably not needed here

**Review by jahnavi1satya — CHANGES_REQUESTED** [review]

File: saucelabs.conf.js (hunk 2 above)
Comment (elizereyes123) [reply saucelabs.conf.js]: I've updated the saucelabs.conf.js to remove these

File: src/test/end-to-end/codecept.conf.js (hunk 1 above)
Comment (elizereyes123) [reply src/test/end-to-end/codecept.conf.js]: I've updated the codecept.conf.js to remove these

**Review by jahnavi1satya — APPROVED** [review]

**Review by shashisk97 — APPROVED** [review]
LGTM

**Pull request merged**